src/main/java/Entities/Song.java

```diff
@@ -1,4 +1,60 @@
 package Entities;
+import java.awt.image.BufferedImage;
+import java.util.List;
 
 public class Song {
+
+    private final int id;
+    private final String name;
+    private final List artistList;
+    private final int length;
+    private final String genre;
+    private final String fileLocation;
+    private final boolean isExplicit;
+    private final BufferedImage cover;
+
+
+    public Song(int id, String name, List artistList, int length, String genre,
+                String fileLocation, boolean isExplicit, BufferedImage cover){
+        this.id = id;
+        this.name = name;
+        this.artistList = artistList;
+        this.length = length;
+        this.genre = genre;
+        this.fileLocation = fileLocation;
+        this.isExplicit = isExplicit;
+        this.cover = cover;
+    }
+
+    public int getID(){
+        return this.id;
+    }
+
+    public String getName(){
+        return this.name;
+    }
+
+    public List getArtistList(){
+        return this.artistList;
+    }
+
+    public int getLength(){
+        return this.length;
+    }
+
+    public String getGenre(){
```

Comment:
I don't believe we need getGenre() anymore, as we're not having genre-related spaces. (Unless playlists require genre, but I don't think they do.)

Reply:
Hi. Pls don't remove genre info, I use it for recommendations.

Reply:
I figured it'd be nice as a label next to songs also (like how length isn't really needed, either). You guys might not end up displaying it but I figure its better to have the option.

Reply:
Oops! Never mind then.